lya_cosmo/py/CAMB_model.py

```diff
@@ -0,0 +1,112 @@
+import camb_cosmo
+import likelihood_parameter
+import copy
+import fit_linP
+import camb
+import numpy as np
+
+class CAMBModel(object):
+    """ Container for a set of likelihood parameters describing a CAMB
+    cosmology """
+
+    def __init__(self,zs,cosmo=None,like_params=None,kp_Mpc=0.69,verbose=False):
+
+        self.zs=zs
+        self.kp_Mpc=kp_Mpc
+        self.camb_params=["ombh2","omch2","As","ns"] ## List of parameters we'll use
+                                                     ## to define a cosmology
```

Comment:
We certainly want more, I thought it would be worth discussing or at least thinking some more about which parameters we want to include, but wanted to get the code working first so just picked a small number of the most obvious parameters.

Reply:
Soundgs good. I just thought it would make more sense to keep ombh2 and omch2 fixed, and change H_0 or Omega_m instead. But let's start with this, and then iterate.